### PR TITLE
Updated sensor.py to include ability to status entry sensors

### PR DESCRIPTION
**Pull request by qpe123**
Updated sensor.py to include ability to status entry sensors

### CODE REVIEW

File: simplipy/sensor.py [hunk 1]
```diff
@@ -74,6 +74,13 @@ def low_battery(self) -> bool:
     def settings(self) -> bool:
         """Return the sensor's settings."""
         return self.sensor_data['setting']
+    
+    @property
+    def entrystatus(self) -> str:
```
Comment (bachya) [inline simplipy/sensor.py]: Let's use `entry_status` as a property name.

File: simplipy/sensor.py [hunk 2]
```diff
@@ -74,6 +74,13 @@ def low_battery(self) -> bool:
...
+        if self._type == SensorTypes.entry:
```
Comment (bachya) [inline simplipy/sensor.py]: Please follow [the same pattern that is used for temperature sensors in V3 systems](https://github.com/bachya/simplisafe-python/blob/01fdd25d28d9f1decab7011ec7499982d7e69f0c/simplipy/sensor.py#L118-L125).

**Comment by bachya** [comment]
While investigating further, I don't think this is the correct approach. The goal should be to provide an as-common-as-possible interface between V2 and V3 systems.

* V2 entry sensors have an `entryStatus` property that can be `open` or `closed`.
* V3 systems utilize the `status.triggered` property chain to communicate the same information (but with values of `true` or `false`).

Since this PR is well behind the dev base and is nearing a week of inactivity, I'm going to close it and apply these principles via a new PR. Thanks for your assistance!

**Comment by qpe123** [comment]
Yeah I apologize and didn't get a chance to redo the merge as I have been
traveling... I only have a v2 system so couldn't determine the call for v3,
but sounds like you got it covered so no issue at all with you moving it
forward.

Thanks,
Quinn

**Pull request closed without merging**